<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionMenu</class>
 <widget class="QDialog" name="ConnectionMenu">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>345</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>371</width>
    <height>345</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>371</width>
    <height>345</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="aliasLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Alias</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="aliasEdit">
     <property name="toolTip">
      <string>Name the Server goes By</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="addressLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Server Address</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="addressEdit">
     <property name="toolTip">
      <string>Address of the server</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="portLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Server Port</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="portEdit">
     <property name="toolTip">
      <string>Port to connect to</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="connectStrLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Connection Str.</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="connectStrEdit">
     <property name="toolTip">
      <string>Special string for use when connecting to server</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QCheckBox" name="connectStartupCheck">
     <property name="toolTip">
      <string>Should this server connect upon startup?</string>
     </property>
     <property name="text">
      <string>Connect on Startup</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QCheckBox" name="sendReportCheck">
     <property name="toolTip">
      <string>Should this server connect upon startup?</string>
     </property>
     <property name="text">
      <string>Send Position Reports</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QCheckBox" name="authConnCheck">
     <property name="toolTip">
      <string>Should this server connect upon startup?</string>
     </property>
     <property name="text">
      <string>Authenticate Connection</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>aliasEdit</tabstop>
  <tabstop>addressEdit</tabstop>
  <tabstop>portEdit</tabstop>
  <tabstop>connectStrEdit</tabstop>
  <tabstop>connectStartupCheck</tabstop>
  <tabstop>sendReportCheck</tabstop>
  <tabstop>authConnCheck</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConnectionMenu</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConnectionMenu</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
